physics-integration-layer/src/data_for_backends.rs

```diff
@@ -218,3 +218,12 @@ impl TnuaGhostSensor {
 /// See `TnuaSimpleFallThroughPlatformsHelper`.
 #[derive(Component, Default, Debug)]
 pub struct TnuaGhostPlatform;
+
+#[derive(Component, Debug, Clone, Copy)]
+pub struct TnuaGravity(pub Vector3);
+
+impl Default for TnuaGravity {
+    fn default() -> Self {
+        TnuaGravity(Vector3::ZERO)
```

Comment:
`TnuaGravity` should not be `Default` either - and even if it was, the default should be the Earth's gravity rather than zero gravity. Though that's, too, is debatable - which is one of the reasons it shouldn't be `Default`.